### PR TITLE
Event checkin issue 303.

### PR DESCRIPTION
**Pull request by karbmk**
Files changed:
1. views.py
2. live-dashboard.html

### CODE REVIEW

File: openCurrents/templates/live-dashboard.html [hunk 1]
```diff
@@ -124,12 +124,12 @@ <h6 class="left half-margin-top half-margin-bottom">Registered volunteers</h6>
                   </div>
 
                   <div class="right small-3 columns">
-                    <input {% if checkin_buttons %}disabled{% endif %} type="checkbox" id="vol-checkin-{{ user.id }}" name="vol-checkin-{{ user.id }}" value="{{ user.id }}" class=" hidden checkin-checkbox"
+                    <input {% if checkin_disabled %}disabled{% endif %} type="checkbox" id="vol-checkin-{{ user.id }}" name="vol-checkin-{{ user.id }}" value="{{ user.id }}" class=" hidden checkin-checkbox"
```
Comment (nickolashe) [inline openCurrents/templates/live-dashboard.html]: @karbmk @tsiology lets try using intuitive variable names

File: openCurrents/views.py [hunk 2]
```diff
@@ -1364,26 +1365,28 @@ def get_context_data(self, **kwargs):
...
-            context['checkin_buttons'] = False
-        print(context['checkin_buttons'])
-        event_orgid = event.project.org.id
         context['event'] = event
-        # verify user is part of admin_<event_orgid> group
-        if not self.request.user.groups.filter(name='admin_'+str(event_orgid)).exists():
-            context['forbidden'] = True
+
+        # disable checkin if event is too far in future
+        if event.datetime_start > datetime.now(tz=pytz.UTC) + timedelta(minutes=15):
```
Comment (nickolashe) [inline openCurrents/views.py]: allow checkin beginning 15 min prior to event

**Comment by nickolashe** [comment]
@karbmk I am unable to add new volunteers from live-dashboard. Can you confirm you are seeing the same please?

**Comment by karbmk** [comment]
Yes I did see that. It was because it was not retuning HTTP response back to the Ajax template in UI. I think it's fixed now. can you check once.

**Comment by dannypernik** [comment]
@karbmk @nickolashe safe to delete branch?

**Pull request merged**